src/sagemaker/workflow/steps.py

```diff
@@ -218,10 +224,14 @@ def __init__(
             name (str): The name of the transform step.
             transformer (Transformer): A `sagemaker.transformer.Transformer` instance.
             inputs (TransformInput): A `sagemaker.inputs.TransformInput` instance.
+            experiment_config (Dict[str, str]): A dictionary describing the experiment configuration
+                for the transform job. Dictionary contains three optional keys,
+                'ExperimentName', 'TrialName', and 'TrialComponentDisplayName'. Default: to `None`.
```

Comment:
```suggestion
                'ExperimentName', 'TrialName', and 'TrialComponentDisplayName'. Defaults: to `None`.
```

Reply:
Changed to "Defaults to" as per other comments.